component-library/components/sections/Left-right-block--Featured-map/Left-right-block--Featured-map.bookshop.yml

```
@@ -0,0 +1,80 @@
+# Metadata about this component, to be used in the CMS
+spec:
+  structures:
+    - content_blocks
+  label: "Contact block"
+  description: Contact content block
+  icon: contact_phone
+  tags: ['contact', 'content-block', 'left-right']
+
+# Defines the structure of this component, as well as the default values
+blueprint:
+  content:
+    social_icons: true
+    heading: 
+      primary_heading: A contact block
+    description: This is a simple yet powerful block to display not only your location by using Google Maps, but also any other contact details you wish to share with your customers.
+    contact_details:
+      - text: 'Email address: mybusiness@business.com'
+        hero_icon_name: envelope
+    button: bookshop:generic/button!    
+  styles:
+    orientation: Left
```

```suggestion
    map_alignment: Left
```
Otherwise orientation: left doesn't actually tell me what's left